metricbeat/mb/testing/data_generator.go

```diff
@@ -141,17 +142,17 @@ func StandardizeEvent(ms mb.MetricSet, e mb.Event, modifiers ...mb.EventModifier
 // WriteEventToDataJSON writes the given event as "pretty" JSON to
 // a ./_meta/data.json file. If the -data CLI flag is unset or false then the
 // method is a no-op.
-func WriteEventToDataJSON(t testing.TB, fullEvent beat.Event, pathPostfix string) {
+func WriteEventToDataJSON(t testing.TB, fullEvent beat.Event, postfixPath string) {
```

Comment:
@jsoriano Does that look like discussed?

Reply:
Yes :)